index.html

```diff
@@ -159,7 +178,10 @@
 
 		<div class="hero">
 			<div style="text-align:center;">	
-				<img src="/assets/celebration.svg">
+				<picture>
+					<source srcset="/assets/celebration-dark.svg" alt="celebration" media="(prefers-color-scheme: dark)">
+					<img src="/assets/celebration.svg" alt="celebration" width="427" height="231">
```

Comment:
The `alt` attribute should be a human readable description of the image. Please provide more details than "celebration"